web-ts/src/index.ejs

```diff
@@ -0,0 +1,57 @@
+<!doctype html>
```

Comment:
Is this already our `index.ejs` file? Lots of stuff removed, could we just comment it out?

Reply:
This is our ejs file without our variables as it was difficult to get it working. I think we should add a BIG BOLD COMMENT here that it is not finished.